From 97aa7ba618f29f5c90165860ff4cf2c79bd32a2e Mon Sep 17 00:00:00 2001
From: mrtnvgr <martynovegorOF@yandex.ru>
Date: Fri, 30 Dec 2022 13:02:48 +0700
Subject: [PATCH] background alpha xresources support

---
 x.c | 1 +
 1 file changed, 1 insertion(+)

diff --git a/x.c b/x.c
index 2eb3583..4e9b3f9 100644
--- a/x.c
+++ b/x.c
@@ -2154,6 +2154,7 @@ xrdb_load(void)
 
 		XRESOURCE_LOAD_FLOAT("cwscale", cwscale);
 		XRESOURCE_LOAD_FLOAT("chscale", chscale);
+		XRESOURCE_LOAD_FLOAT("backgroundAlpha", backgroundAlpha);
 
 		XRESOURCE_LOAD_INTEGER("tabspaces", tabspaces);
 	}
-- 
2.39.0

