From d4388e8e7c7174c48a2f763e6223c9d96e339e50 Mon Sep 17 00:00:00 2001
From: mrtnvgr <martynovegorOF@yandex.ru>
Date: Wed, 17 May 2023 13:32:56 +0700
Subject: [PATCH] osc all colors reset: add support for xresources

---
 st.c | 1 +
 st.h | 2 ++
 2 files changed, 3 insertions(+)

diff --git a/st.c b/st.c
index dea0ea6..9e1496a 100644
--- a/st.c
+++ b/st.c
@@ -2019,6 +2019,7 @@ strhandle(void)
 			p = strescseq.args[2];
 			/* FALLTHROUGH */
 		case 111: /* all colors reset */
+			xrdb_load();
 			xloadcols();
 			tfulldirt();
 			return;
diff --git a/st.h b/st.h
index 70e4c8b..3929130 100644
--- a/st.h
+++ b/st.h
@@ -78,6 +78,8 @@ typedef union {
 	const char *s;
 } Arg;
 
+void xrdb_load(void);
+
 void die(const char *, ...);
 void redraw(void);
 void draw(void);
-- 
2.40.1

