From 4fab2d271caaa87d08e3749c7bef9dedd3832a9f Mon Sep 17 00:00:00 2001
From: mrtnvgr <martynovegorOF@yandex.ru>
Date: Fri, 30 Dec 2022 13:17:07 +0700
Subject: [PATCH] boxdraw xresources support

---
 config.def.h   |   6 +++---
 st.h           |   2 +-
 x.c            |   3 +++
 3 files changed, 8 insertions(+), 4 deletions(-)

diff --git a/config.def.h b/config.def.h
index 4642306..62275f7 100644
--- a/config.def.h
+++ b/config.def.h
@@ -70,11 +70,11 @@ static unsigned int cursorthickness = 2;
  *    Bold affects lines thickness if boxdraw_bold is not 0. Italic is ignored.
  * 0: disable (render all U25XX glyphs normally from the font).
  */
-const int boxdraw = 1;
-const int boxdraw_bold = 1;
+int boxdraw = 1;
+int boxdraw_bold = 1;
 
 /* braille (U28XX):  1: render as adjacent "pixels",  0: use font */
-const int boxdraw_braille = 1;
+int boxdraw_braille = 1;
 
 /*
  * bell volume. It must be a value between -100 and 100. Use 0 for disabling
diff --git a/st.h b/st.h
index 2f7b289..70e4c8b 100644
--- a/st.h
+++ b/st.h
@@ -135,5 +135,5 @@ extern float backgroundAlpha;
 extern unsigned int defaultfg;
 extern unsigned int defaultbg;
 extern unsigned int defaultcs;
-extern const int boxdraw, boxdraw_bold, boxdraw_braille;
+extern int boxdraw, boxdraw_bold, boxdraw_braille;
 extern double erreschide;
diff --git a/x.c b/x.c
index 8e6a1bf..1f8f0cc 100644
--- a/x.c
+++ b/x.c
@@ -2168,6 +2168,9 @@ xrdb_load(void)
 		XRESOURCE_LOAD_FLOAT("backgroundAlpha", backgroundAlpha);
 
 		XRESOURCE_LOAD_INTEGER("tabspaces", tabspaces);
+		XRESOURCE_LOAD_INTEGER("boxdraw", boxdraw);
+		XRESOURCE_LOAD_INTEGER("boxdraw_bold", boxdraw_bold);
+		XRESOURCE_LOAD_INTEGER("boxdraw_braille", boxdraw_braille);
 	}
 	XFlush(dpy);
 }
-- 
2.39.0

